OnBoardingCalculator/app/src/main/java/com/globant/onboardingcalculator/mvp/model/CalculatorModel.java

```diff
@@ -36,25 +38,19 @@ public void setOperator(char operator) {
     }
 
     public void setFirstOperand(String firstOperand) {
-        if (this.firstOperand.isEmpty())
-            this.firstOperand = firstOperand;
-        else
-            this.firstOperand = this.firstOperand + firstOperand;
+        this.firstOperand = firstOperand;
     }
 
     public void setSecondOperand(String secondOperand) {
-        if (this.secondOperand.isEmpty())
-            this.secondOperand = secondOperand;
-        else
-            this.secondOperand = this.secondOperand + secondOperand;
+        this.secondOperand = secondOperand;
     }
 
-    public void digitDeletedInFirstOperand(String newOperand){
-        firstOperand = newOperand;
-    }
-
-    public void digitDeletedInSecondOperand(String newOperand){
-        secondOperand = newOperand;
+    public void digitDeletedInOperand(String operand) {
```

Comment:
this logic will break if you firstOperand and secondOperand are the same

Reply:
Done!